.github/workflows/test.yml

```
@@ -492,26 +442,22 @@ jobs:
         run: npm install
 
       - name: Install nextest
-        id: nextest-install
-        continue-on-error: true
-        run: curl -LsSf https://get.nexte.st/latest/${{ matrix.os.nextest }} | tar zxf - -C ${CARGO_HOME:-~/.cargo}/bin
-        shell: bash
+        uses: taiki-e/install-action@nextest
 
-      - name: Install nextest (second try)
-        if: steps.nextest-install.outcome == 'failure'
-        run: curl -LsSf https://get.nexte.st/latest/${{ matrix.os.nextest }} | tar zxf - -C ${CARGO_HOME:-~/.cargo}/bin
-        shell: bash
+      - name: Build nextest
+        timeout-minutes: 120
+        run: |
+          # We exclude turbo as it requires linking Go and all logic resides in turborepo-lib
+          cargo nextest run --no-run --workspace --release --exclude turbo
```

```suggestion
        # We exclude turbo as it requires linking Go and all logic resides in turborepo-lib
        run: |
          cargo nextest run --no-run --workspace --release --exclude turbo
```

This should be a comment in the workflow file and not part of the command.